drivers/tmp006/tmp006.c

```diff
@@ -237,6 +255,11 @@ int tmp006_read_temperature(const tmp006_t *dev, int16_t *ta, int16_t *to)
     tmp006_convert(rawvolt, rawtemp,  &tamb, &tobj);
     *ta = (int16_t)(tamb*100);
     *to = (int16_t)(tobj*100);
+#endif   
+
+    if (tmp006_set_standby(dev)) {
```

Comment:
This needs to be commented in  the commit message. It seems like you added automatic handling of low power mode.

Reply:
Perhaps this can be placed either inside a preprocessor if, and the dependency on xtimer made conditional, or a parameter added to the TMP006_RAW_VALUES struct to be able to choose at run time.